docs/src/main/paradox/server-side/low-level-api.md

```diff
@@ -218,6 +218,9 @@ Java
 
 The third type of failure that can occur is when the connection has been properly established,
 however afterwards is terminated abruptly – for example by the client aborting the underlying TCP connection.
+(Note that this is when the TCP connection is abruptly terminated in an orderly fasion, if the client just goes away,
```

Comment:
Full new sentence in () is a bit weird, just a normal one is fine

Reply:
`abruptly` sounds at odds to me to `orderly fasion` (note also typo, `fashion`), this needs more wording to explain properly I feel

Reply:
The reason I didn't do a regular sentence is that then the following sentence becomes weird.

Reply:
As it refers back to the clean termination sentence.